sample/src/main/AndroidManifest.xml

```diff
@@ -41,6 +41,29 @@
             android:name=".MainActivity"
             android:label="@string/app_name"
             android:theme="@style/AppTheme.NoActionBar">
+            <!--<intent-filter>-->
```

Comment:
Personal preference but wouldn't it be better to make one of the MainActivitys (Java vs Kotlin) a secondary activity instead of a main/launcher activity? Then allowing the user to switch between them through the UI instead of making changes to the manifest and recompiling the app. IMO it would allow users to play with either language's syntax seamlessly.